lib/rubocop/cop/rspec/spec_file_path_format.rb

```diff
@@ -59,6 +65,53 @@ def on_top_level_example_group(node)
 
         private
 
+        # Inflector module that uses ActiveSupport for advanced inflection rules
+        module ActiveSupportInflector
+          def self.call(string)
+            ActiveSupport::Inflector.underscore(string)
+          end
+
+          def self.prepare_availability(config)
```

Comment:
Cop_config may differ across directories.
Does it make sense to cache the preparations?